apcd-cms/src/apps/admin_regis_table/static/admin_regis_table/registrations_table.css

```diff
@@ -1,31 +1,32 @@
 .registration-table {
-    margin-right: auto;
-    text-align: center;
-    column-width: auto;
```

Comment:
@edmondsgarrett These styles did not appear to do anything. If they did have a purpose, please share so I can make sure I didn't break something.

<details>

- The `column-width` default value is `auto`. I did **not** know about this property though, so I am happy to see it used. I'm always learning.
- The `margin-right` may have been useful for table when it did not stretch.

</details>